addons/web-support/impl/src/main/java/org/jboss/windup/web/addons/websupport/tsmodelgen/TypeScriptModelsGenerator.java

```diff
@@ -269,6 +270,31 @@ private ModelDescriptor createModelDescriptor(Class<? extends WindupVertexFrame>
                 modelRelation.methodsPresent.addAll(methodInfo.methodsPresent);
                 modelDescriptor.addRelation(modelRelation);
             }
+
+            setInProperties:
+            {
```

Comment:
That's something I'll do later I think. The method started quite small and got bloated like this over time. It needs some refactoring...